chipy_org/apps/main/views.py

```diff
@@ -37,9 +37,7 @@ def get_context_data(self, **kwargs):
         context.update(kwargs)
 
         context["other_meetings"] = self.get_non_main_meetings(num=3)
-        context["general_sponsors"] = GeneralSponsor.objects.all().order_by(
-            "sponsor__sponsor_group__list_priority", "sponsor__name"
-        )
+        context["sponsor_groups"] = SponsorGroup.objects.prefetch_related("sponsors")
```

Comment:
Nice work overall! 

We sort of want something analogous to the old lines 40 to 41 that you deleted. That is, we want ordering by 'list_priority' from the `SponsorGroup` and then ordering by 'name' of the `Sponsor`.
............................................................................................................

1)Please change new line 40 to become:
`context["sponsor_groups"] = SponsorGroup.objects.prefetch_related("sponsors").order_by("list_priority")`

2)Then, `make format` and` make lint` since that line will be too long.

After ordering by 'list_priority' of the` SponsorGroup`, you will NOT need to change anything else to order by the 'name' of the `Sponsor`.  This is because that's already handled by the 'meta' class in the `Sponsor` model. (See chipy_org/apps/sponsors/models.py , lines 59-60 )

Reply:
Just added the list priority to the sponsor groups as well to resolve the issue.  Also cleaned up tests a little like we did for the job board.  Good catch.  Thanks!